docs/source/conf.py

```diff
@@ -44,6 +45,20 @@ def get_authors_from_citation_file(file_name):
     return author[:-2]
 
 
+def get_python_version_from_pyproject():
+    """
+    Read python version from pyproject.toml file
+
+    """
+    with open("../../pyproject.toml") as file:
+        pyproject = toml.load(file)
+
+    try:
+        return pyproject['project']['requires-python'].replace(">", "").replace("=", "")
+    except KeyError:
```

Comment:
I think you shouldn't catch the keyerror here, since not finding this key means you have broken your package configuration, you want to get an error.

Also replacing the comparison operators is not a good idea I think. We just insert the whole version spec into the docs:

`ctapipe currently requires python: {requires-python}`